Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/SingleCreditCardView.swift

```diff
@@ -233,11 +240,11 @@ class SingleCreditCardViewController: UIViewController, UITableViewDelegate, UIT
 
     private func creditCardCell(indexPath: IndexPath) -> UITableViewCell {
         guard let hostingCell = cardTableView.dequeueReusableCell(
-            withIdentifier: HostingTableViewCell<CreditCardItemRow>.cellIdentifier) as? HostingTableViewCell<CreditCardItemRow> else {
+            withIdentifier: HostingTableViewCell<CreditCardItemRow>.cellIdentifier) as? HostingTableViewCell<CreditCardItemRow>,
+              let creditCard = viewModel.state == .save ? viewModel.decryptedCreditCard?.toFakeCreditCard() : viewModel.creditCard else {
```

Comment:
:warning: Note: The line is way too long, is it possible to refactor into a more readable condition? 
```suggestion
              let creditCard = viewModel.state == .save ? viewModel.decryptedCreditCard?.toFakeCreditCard() : viewModel.creditCard else {
```